### DISCUSSION TITLE
Master key expiration?

### DISCUSSION THREAD

**Discussion by morrogenduath** [keepassxreboot/keepassxc#6161] [Q&A] [answered]
The [original KeePass documentation mentions](https://keepass.info/help/base/keys.html) a master key expiration reminder.
After x amount of time when opening the database, there's a pop-up notifies you that the master key hasn't been changed and advises you to change it.

Does KeePassXC have this functionality?

**Comment by phoerious** [comment, marked as answer]
No, it doesn't and I don't really see the need for it. Changing passwords regularly is a bad habit that leads to simple passwords that are easy to memorise, but also easy to crack. Instead of changing your master password every x months, you should be using a strong passphrase and ideally some sort of second factor, such as a YubiKey.
